### ISSUE TITLE
Add ZORALab's Hestia Credits

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#18]
# Description
```
Please provide a short description of what feature you're looking forward to
add into the existing repository below. Please include the story behind your
idea as well to give a better visualization of your idea.
```
Since we support ZORALab's Hestia as a vendor, we should add its accrediation.

# Expected Behavior
```
Please specify the expected behavior of your requested feature. Some great and
helpful pointers are your expected interface (e.g. command patterns, simple
sketches of the user interface, etc).
```
ZORALab's Hestia meta tag is shown.

# Current Absent Behavior
```
Please specify the current behavior (e.g. workaround, blockage, etc).
```
ZORALab's Hestia meta tag is missing.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Implemented in 674dea0d5257d753838b8cd9d66b4df6e05fcadf

**Comment by hollowaykeanho** [comment]
Released in https://github.com/ZORALab/Hestia/releases/tag/v1.0.0
